cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -80,6 +82,12 @@ public class GenerateCommand extends CommandCommons {
   @Option(names = { "--templates", "-t" }, split = ",", description = MessagesConstants.TEMPLATES_OPTION_DESCRIPTION)
   List<String> templates = null;
 
+  /**
+   * Enforces usage of the old monolithic template structure instead of the new template sets structure.
+   */
+  @Option(names = { "--force-monolithic-templates" }, description = MessagesConstants.FORCE_MONOLITHIC_TEMPLATES)
```

Comment:
Could we rename this and all occurrences and talk about monolithic configuration rather then monolithic templates?
In essence the configuration is monolithic and not the templates.